File: app/src/main/java/com/bunbeauty/ideal/myapplication/cleanArchitecture/models/entity/Photo.java
package com.bunbeauty.ideal.myapplication.cleanArchitecture.models.entity;

public class Photo {

    private String photoId;
    private String photoLink;
    private String photoOwnerId;

    public String getPhotoId() {
        return photoId;
    }
    public void setPhotoId(String _photoId){
        photoId = _photoId;
    }

    public String getPhotoLink() {
        return photoLink;
    }
    public void setPhotoLink(String _photoLink){
        photoLink = _photoLink;
    }

    public String getPhotoOwnerId() {
        return photoOwnerId;
    }
    public void setPhotoOwnerId(String _photoOwnerId){
        photoOwnerId = _photoOwnerId;
    }
}

File: app/src/main/java/com/bunbeauty/ideal/myapplication/cleanArchitecture/models/entity/Photo.kt
package com.bunbeauty.ideal.myapplication.cleanArchitecture.models.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class Photo(@PrimaryKey
            var id: String = "",
            var link: String = "",
            var ownerId: String = "") {

    lateinit var userId: String
    lateinit var serviceId: String
    companion object {
        const val PHOTOS = "photos"
        const val PHOTO_LINK = "photo link"
        const val OWNER_ID = "owner id"
    }

}
